### PR TITLE
fix expect import

### PR DESCRIPTION
**Pull request by lacell75**
Fix error : `TypeError: expect.stringContaining is not a function`

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import isEqual from 'lodash.isequal'
 import type { ParsedCSSValue } from 'webdriverio'
-import expect from 'expect'
+
+import { expect } from 'expect-webdriverio'
```
Comment (christian-bromann) [inline src/utils.ts:4]: Can you verify we don't have any circular dependencies?
```suggestion
import { expect } from './index.js'
```

Comment (lacell75) [reply src/utils.ts]: sure, fixed

**Review by christian-bromann — APPROVED** [review]
Thanks a lot 👍

**Comment by lacell75** [comment]
Could you release a 4.15.x with this fix ?

Le lun. 19 août 2024 à 21:00, WebdriverIO Bot ***@***.***> a
écrit :

**Comment by christian-bromann** [comment]
released in [`v4.15.4`](https://www.npmjs.com/package/expect-webdriverio/v/4.15.4)

**Comment by lacell75** [comment]
Thanks!

**Pull request merged**